core/src/main/scala/quasar/destination/gbq/GBQFlow.scala

```diff
@@ -255,7 +255,7 @@ final class GBQFlow[F[_]: Concurrent](
                 .as(loc.uri.asRight[InitializationError[Json]])
           }
           case otherStatus =>  
-            resp.attemptAs[String].fold(
+            resp.attemptAs[String].foldF(
```

Comment:
Did you see an error being logged of the "with-body" form? 

Maybe while we are at it, we can also change the text here a bit because it's the response rather than the body that is tried to be decoded.

E.g.:

`err => Sync[F].delay(log.error(s"GBQ job creation failed with status '$otherStatus'. Failed to decode response: ${err.message}"`

`r => Sync[F].delay(log.error(s"GBQ job creation failed with response: $r")))`


Similarly, I would change the `body` of line 165 in `r` / `respString` or something

Reply:
> Did you see an error being logged of the "with-body" form?

Yup

```
2021-09-02 13:18:54,340 ERROR [sdbe-compute-18] (quasar.destination.gbq.GBQFlow) - GBQ job creation failed with status '404 Not Found': {
  "error": {
    "code": 404,
    "message": "Not found: Dataset precog-ci-275718:domas-test",
    "errors": [
      {
        "message": "Not found: Dataset precog-ci-275718:domas-test",
        "domain": "global",
        "reason": "notFound"
      }
    ],
    "status": "NOT_FOUND"
  }
}
```

I did the other renames you asked for